src/cmd/manage_nsfs.js

```diff
@@ -0,0 +1,411 @@
+            }],
+            nsfs_account_config: {
+                uid: uid,
+                gid: gid,
+                new_buckets_path: new_buckets_path
+            }
+        };
+    }
+    if (action === 'update') {
+        data = _.omitBy(data, _.isUndefined);
+        data = update_account(config_root, data);
+    }
+    return data;
+}
+
+async function update_account(config_root, target) {
+    const account_path = config_root + '/accounts';
+    let source;
+    try {
+        const source_raw = await fs.promises.readFile(path.join(account_path, target.access_keys[0].access_key + '.json'));
+        source = JSON.parse(source_raw.toString());
+        target.nsfs_account_config.new_buckets_path = target.nsfs_account_config.new_buckets_path ||
+                                                        source.nsfs_account_config.new_buckets_path;
+    } catch (err) {
+        console.error('NSFS Manage command: Could not find account ' + target.access_keys[0].access_key + ' to update');
+        print_account_usage();
+    }
+        const data = _.merge({}, source, target);
+    return data;
+}
+
+async function manage_account_operations(action, data, config_root) {
+    const account_path = config_root + '/accounts';
+    if (action === 'add') {
+        const is_valid = await validate_account_add_args(data);
+        if (!is_valid) {
+            print_account_usage();
+            return;
+        }
+        create_resourse(data.access_keys[0].access_key, account_path, data);
+    } else if (action === 'status') {
+        const is_valid = await validate_minimum_account_args(config_root, data);
+        if (!is_valid) {
+            print_account_usage();
+            return;
+        }
+        const bucket_dir_stat = await fs_utils.file_exists(path.join(account_path, data.access_keys[0].access_key + '.json'));
+        if (bucket_dir_stat) {
+            console.log("Account exists with access key : " + data.access_keys[0].access_key);
+        } else {
+            console.log("Account do not exists with access key : " + data.access_keys[0].access_key);
+        }
+    } else if (action === 'update') {
+        const is_valid = await validate_account_add_args(data);
+        if (!is_valid) {
+            print_account_usage();
+            return;
+        }
+        create_resourse(data.access_keys[0].access_key, account_path, data);
+    } else if (action === 'delete') {
+        const is_valid = await validate_minimum_account_args(config_root, data);
+        if (!is_valid) {
+            print_account_usage();
+            return;
+        }
+        delete_resourse(data.access_keys[0].access_key, account_path);
+    } else if (action === 'list') {
+        const accounts = await list_resources(account_path);
+        console.log(accounts);
+    } else {
+        console.log("Account action not found.");
+        print_account_usage();
+    }
+}
+
+async function create_resourse(file_name, file_path, data) {
```

Comment:
I would suggest to change the create_resource to create_config_file(), resource has a different meaning in NooBaa and I find it a bit confusing